test/unit/common/persistence/sql/PlatformTest.php

```diff
@@ -23,7 +23,14 @@
 
 class PlatformTest extends TestCase
 {
-
+    protected function setUp()
+    {
+        if (!extension_loaded('pdo_sqlite')) {
```

Comment:
You can just use oat\generis\test::getSqlMock() inside test cases below. It does the same (checking the pdo_sqlite is loaded and creating connection. With this approach you don't ever need to have this setUp().